app/controllers/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @users = User.all
+    render json: @users
```

Comment:
Do not forget to add pagination here or move it to the next PR

Reply:
resolved